test/functional/cmdLineTests/lockWordAlignment/alignmentSettings.mk

```diff
@@ -21,10 +21,10 @@
 ##############################################################################
 
 ADD_EXPORTS=
-ASM_JAR=$(LIB_DIR)$(D)asm-all.jar
+ALLOW_ILLEGAL_ACCESS=
 # ADD_EXPORTS needs to set for JDK9 and up
 # if JDK_VERSION is not 8
 ifeq ($(filter 8, $(JDK_VERSION)),)
  ADD_EXPORTS=--add-exports=java.base/com.ibm.oti.vm=ALL-UNNAMED
- ASM_JAR=$(LIB_DIR)$(D)asm.jar
-endif
\ No newline at end of file
+ ALLOW_ILLEGAL_ACCESS=--illegal-access=permit
```

Comment:
Can this use --add-exports instead, not sure the `--illegal-access` option will be available forever.

Reply:
```--illegal-access``` has been replaced with ```--add-exports```, thanks.